src/Api/HttpApi.php

```diff
@@ -30,7 +29,7 @@ abstract class HttpApi
     /**
      * The HTTP client.
      *
-     * @var ClientInterface|PluginClient
+     * @var ClientInterface
```

Comment:
In case anyone reviewing this PR is confused, `PluginClient` implements `HttpClient` which implements `ClientInterface`, so the type union here was redundant, and it makes sense to simplify the type here to just `ClientInterface`.